core/src/main/scala/com/pingcap/tispark/TiBatchWrite.scala

```diff
@@ -324,7 +348,7 @@ class TiBatchWrite(@transient val df: DataFrame,
         s"invalid transaction tso with startTs=$startTs, commitTs=$commitTs"
       )
     }
-    val commitPrimaryBackoff = ConcreteBackOffer.newCustomBackOff(BackOffer.BATCH_COMMIT_BACKOFF)
+    val commitPrimaryBackoff = ConcreteBackOffer.newCustomBackOff(PRIMARY_KEY_COMMIT_BACKOFF)
```

Comment:
Please keep in mind that it does not mean backOff will be limited in PRIMARY_KEY_COMMIT_BACKOFF. It is still possible that commit duration exceeds PRIMARY_KEY_COMMIT_BACKOFF because we can only check the time duration when we receive a response.